File: C++/02-String-Basics/8.Remove Spaces From String.cpp
// Remove Spaces From String
// text = "I love programming"
// output ="Iloveprogramming"

#include <iostream>
#include <string>

using namespace std;

int main() {

    string text = "I love programming";

    string result = "";

    for (int i = 0; i < text.length(); i++) {

        if (text[i] != ' ') {
            result += text[i];
        }

    }

    cout << result;

    return 0;
}

File: Python/2.String-Basics/8.Remove Spaces From String.py
# Remove Spaces From String 
# Example text= "I love Programming"
# output = IloveProgramming"

def Remove(text):
    if not text:
        return
    
    newText=""
    
    for c in text:
        if c !=" ":
            newText +=c
    return newText

text = "I love Programming"

print(Remove(text))